crates/diman_lib/src/magnitude.rs

```diff
@@ -3,6 +3,8 @@ use core::{
     ops::{Div, Mul},
 };
 
+use crate::num_traits_reexport::*; // either Float or FloatCore
```

Comment:
This is a stylistic choice that I have no attachment to. We could instead use `Float` or `FloatCore` depending on cfg.

Reply:
I think this needs to be 
```rust
#[cfg(feature = "num-traits-libm")]
pub use num_traits::float::Float;

#[cfg(not(feature = "num-traits-libm"))]
pub use num_traits::float::FloatCore;
```

just to get rid of the warning.